lms/static/certificates/sass/_components.scss

```diff
@@ -0,0 +1,641 @@
+// ------------------------------
+// // Open edX Certificates: Components
+
+// About: styling for specific UI components ranging from global to modular.
+
+// #OVERRIDE - COPY
+// #OVERRIDE - HEADINGS
+// #OVERRIDE - BUTTONS
+
+// #DEPTH
+// #MESSAGES
+// #IMAGES
+
+// #HEADER APP
+// #BANNER
+// #INTRODUCTION
+// #ACCOMPLISHMENT
+// #RENDERING
+// #ACCOMPLISHMENT RELATED
+// #CERTIFICATE
+// #ABOUT
+
+// ------------------------------
+// #OVERRIDE - COPY
+// ------------------------------
+// for some reason in Sass compilation, function calls as variables aren't playing well :/
+.copy {
+    color: palette(grayscale, dark);
+
+    &.emphasized {
+        color: palette(grayscale, x-dark);
+        font-weight: font-weight(normal);
+    }
+
+    &.de-emphasized {
+        color: palette(grayscale, base);
+        font-weight: font-weight(normal);
+    }
+}
+
+// ------------------------------
+// #OVERRIDE - HEADINGS
+// ------------------------------
+// for some reason in Sass compilation, function calls as variables aren't playing well :/
+.hd-1,
+.hd-2,
+.hd-3,
+.hd-4,
+.hd-5,
+.hd-6 {
+    font-weight: font-weight(normal);
+    color: palette(grayscale, x-dark);
+
+    &.emphasized {
+        font-weight: font-weight(semi-bold);
+        color: palette(grayscale, x-dark);
+    }
+
+    &.de-emphasized {
+        color: palette(grayscale, base);
+    }
+}
+
+.hd-6,
+.hd-7 {
+    font-weight: font-weight(semi-bold);
+
+    &.emphasized {
+        font-weight: font-weight(bold);
+    }
+}
+
+%hd-subsection {
+    margin-bottom: spacing-vertical(small);
+    border-bottom: rem(2) solid $cert-decorative-color-de-emphasized;
+    padding-bottom: spacing-vertical(xx-small);
+}
+
+// ------------------------------
+// #OVERRIDE - BUTTONS
+// ------------------------------
+a.btn {
+
+    &:hover, &:active, &:focus {
+        border-bottom-color: $transparent;
+    }
+}
+
+.btn-default {
+    background: $transparent;
+
+    // STATE: hover and focus
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(primary, base);
+        color: palette(grayscale, white);
+    }
+}
+
+// ------------------------------
+// #DEPTH
```

Comment:
NOTE: cheating here and doing some PL work ahead of its time with UI depth styling. :P